src/atoms/Button/spec.js

```diff
@@ -1,16 +1,25 @@
 import React from 'react';
 import { shallow } from 'enzyme';
+import cx from 'classnames';
 
 import Button from 'atoms/Button';
 
 const onClick = jest.fn();
 
 describe('Button atom', () => {
   it('should render the button', () => {
-    const wrapper = shallow(<Button onClick={onClick}>Test</Button>);
+    const wrapper = shallow(
+      <Button onClick={onClick} className="test-class">
+        Test
+      </Button>
+    );
     expect(
       wrapper.equals(
-        <button type="button" onClick={onClick}>
+        <button
+          type="button"
+          onClick={onClick}
+          className={cx('button-atom', 'test-class')}
+        >
           Test
         </button>
       )
```

Comment:
Shouldn't the ```expect``` function also be chained with an assertion ``` .toBe(true) ``` ? Is there a default one if it is omitted?

Reply:
Yes that seems accurate. I saw it used with the `.toBe(true)` assertion on a different component (also checking the presence of specific nodes in the `Header.spec.js`) and assumed that it was necessary.